core/metadata.js

```diff
@@ -256,6 +256,9 @@ function assignMaxDate (doc /*: Metadata */, was /*: ?Metadata */) {
 }
 
 const ensureExecutable = (one, two) => {
+  two = process.platform === 'win32'
+    ? _.defaults({executable: one.executable}, two)
```

Comment:
I'm still not sure to understand how it could work, I just wanted to make an experiment and was expecting other cases to break, but it didn't happen...